Makefile

```diff
@@ -75,8 +75,9 @@ migrations:
 	$(manage) makemigrations $(ARGS)
 
 trivy: 	    						## Detect image vulnerabilities
-	$(dc) build app
-	trivy image --ignore-unfixed 127.0.0.1:5001/bereikbaarheid/api
+	$(dc) build --no-cache app
+	trivy image --ignore-unfixed docker-registry.secure.amsterdam.nl/datapunt/bereikbaarheid-backend
```

Comment:
Nu checkt ie dus de bestaande image die al gepushed is. Als je lokaal changes maakt en dan trivy runt dan test hij dus niet de lokale image. Ik zou deze wel op lokaal zetten.

Reply:
jeroen zegt klopt wel :-)